rust/src/data/ffi/mod.rs

```diff
@@ -214,7 +214,7 @@ pub extern "C" fn opendp_data__slice_as_object(
         // https://github.com/pola-rs/pyo3-polars/blob/5150d4ca27c287ff4be5cafef243d9a878a8879d/pyo3-polars/src/lib.rs#L147-L153
         // the slice is lf.__getstate__ from the python side and then deserialized here
         ciborium::de::from_reader(slice).map_err(
-            |e| err!(FFI, "Error when deserializing {}. {}", name, e)
+            |e| err!(FFI, "Error when deserializing {}. This may be because you're using features from Polars that are not currently supported. {}", name, e)
```

Comment:
With #1695 and #1663 in place, mismatched polars versions should be reported earlier. Is there anything _besides_ using unsupported features that we would expect to cause this? If so, do we have an issue to improve the reporting for those cases? If not, then could this message be more confident?